src/com/google/javascript/jscomp/js/es6/util/inherits.js

```diff
@@ -49,25 +49,31 @@
  *
  * @param {!Function} childCtor Child class.
  * @param {!Function} parentCtor Parent class.
+ * @param {boolean} es5Out language out is ES5
  */
-$jscomp.inherits = function(childCtor, parentCtor) {
-  /** @constructor */
-  function tempCtor() {}
-  tempCtor.prototype = parentCtor.prototype;
-  childCtor.superClass_ = parentCtor.prototype;
-  childCtor.prototype = new tempCtor();
-  /** @override */
-    childCtor.prototype.constructor = childCtor;
+$jscomp.inherits = function(childCtor, parentCtor, es5Out) {
+  if (es5Out === true && Object.setPrototypeOf) {
```

Comment:
We can't do that. Because then IE9 won't work at all. IE9 inheritance still works without setPrototypeOf - just not for static methods.

This is a well known restriction and developers are used to this. See https://babeljs.io/docs/usage/caveats/#internet-explorer